composer.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "oomphinc/drupal-scaffold",
+    "description": "A Drupal 8 scaffold project",
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nathan Dentzau",
+            "email": "nathan.dentzau@gmail.com"
+        }
+    ],
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "composer",
+            "url": "https://asset-packagist.org"
+        }
+    ],
+    "require": {
+        "bower-asset/bootstrap": "^4.0",
+        "composer/installers": "^1.5",
+        "cweagans/composer-patches": "^1.6",
+        "drupal-composer/drupal-scaffold": "^2.4",
+        "drupal-composer/preserve-paths": "^0.1.4",
+        "drupal/admin_toolbar": "^1.23",
+        "drupal/coder": "^8.2",
+        "drupal/coffee": "^1.0",
+        "drupal/config_installer": "^1.8",
+        "drupal/config_split": "^1.3",
+        "drupal/core": "^8.5",
+        "drupal/honeypot": "^1.27",
+        "drupal/oomph_paragraphs": "^1.0",
```

Comment:
I don't think this should be in here. It may not be needed for some projects like Genscape.

Reply:
o_O I thought we were standardizing using paragraphs going forward?

Reply:
When we need to create flexible content, but for some sites that kind of content may not be needed. It was identified that Genscape didn't need that kind of content entry.

Reply:
Talked to pico offline. We're going to keep it here and remove it on a per-project basis.